.github/actions/create-github-release-action/README.md

```diff
@@ -0,0 +1,51 @@
+<!--
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2025 The Linux Foundation
+-->
+
+# ⬆️ Create Github Release Action
+
+This action creates a new release in the calling repo using the provided version.
+
+## create-github-release-action
+
+## Usage Example
+
+<!-- markdownlint-disable MD046 -->
```

Comment:
Why do you need these `markdownlint-enable/disable`? Are you planning to add GHA for this repo that check the markdown? I see there are GHAs for fossa and reuse but I do not see anything about markdown

Reply:
We always strive for clean linting, even if checks are not active on the repo (this minimizes tech debt if/when linting enforcement is activated). However, none of these mdl directives were still necessary in the final version of this file, so I've pulled all of them.